Source/interfac.cpp

```diff
@@ -56,7 +56,7 @@ void DrawProgress(int screen_x, int screen_y, int progress_id)
 	BYTE *dst;
 	int i;
 
-	dst = &gpBuffer[screen_x + PitchTbl[screen_y]];

```

Comment:
This is the size of the background you see during the load screen, so they won't change size with the resolution. But maybe they should be centered. I'm also considering just switching logical resolution so that they get stretched to the view size.